lib/moab/file_signature.rb

```diff
@@ -128,12 +128,12 @@ def fixity
     # @param other [FileSignature] The other file signature being compared to this signature
     # @return [Boolean] Returns true if self and other have comparable fixity data.
     def eql?(other)
-      return false unless (other.respond_to?(:size) && other.respond_to?(:checksums))
-      return false if self.size.to_i != other.size.to_i
-      self_checksums = self.checksums
+      return false unless other.respond_to?(:size) && other.respond_to?(:checksums)
+      return false if size.to_i != other.size.to_i
+      self_checksums = checksums
       other_checksums = other.checksums
       matching_keys = self_checksums.keys & other_checksums.keys
-      return false if matching_keys.size == 0
```

Comment:
the intent according to this comment:
https://github.com/sul-dlss/moab-versioning/pull/152/commits/27db28377dd5848764f2a1d98d69146784c96890#diff-11a0d7906801d9dea0eccb85667ad811R75

seemed to be to keep `#size == 0` instead of switching to `#empty`, for compatibility's sake?

Reply:
It only matters when called on `Pathname` objects, which don't have `empty?` on old rubies.  Here, it is called on an Array, and that's fine.